src/Microsoft.AspNetCore.Mvc.Core/Filters/FilterCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Filters
+{
+    public static class FilterCollectionExtensions
+    {
+        public static IFilterMetadata Add<TFilterType>(this FilterCollection collection)
```

Comment:
Good point, didn't think of the `using` statement. Damien also mentioned 'extension methods' so I figured that would be OK.